test/adding_tests.md

```diff
@@ -139,7 +139,7 @@ The `Clients` struct contains initialized clients for accessing:
 For example, to create a `Route`:
 
 ```bash
-_, err = clients.Routes.Create(test.Route(namespaceName, routeName, configName))
+_, err = clients.ServingClient.Routes.Create(test.Route(namespaceName, routeName, configName))
 ```
 
 And you can use the client to clean up `Route` and `Configuration` resources created
```

Comment:
should we add descriptions of 3 client type and when to use which one here?

Reply:
the above paragraph mentions that the clients object provides access to different objects and has examples on using them. Those things should make clear. Also, there is only one build e2e test that uses a build client and is not ususal for other e2e tests.